frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -110,14 +146,36 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
             {infrastructurePlatform}
           </DetailItem>
           <DetailItem title={t('ceph-storage-plugin~Mode')}>Internal</DetailItem>
-          <DetailItem
-            key="version"
-            title={t('ceph-storage-plugin~Version')}
-            isLoading={!subscriptionLoaded}
-            error={subscriptionLoaded && !ocsVersion}
-          >
-            {ocsVersion}
-          </DetailItem>
+          {!isEmpty(filteredVersions) &&
+          ocsSubscription?.[0]?.spec?.installPlanApproval === 'Manual' ? (
+            <DetailItem
+              key="version"
+              title={t('ceph-storage-plugin~Version')}
+              isLoading={!subscriptionLoaded}
+              error={subscriptionLoaded && !ocsVersion}
+            >
+              {ocsVersion}
+              <Button
+                type="button"
+                isInline
+                onClick={ocsChannelModal}
+                variant="link"
+                isDisabled={!ocsSubscriptionLoaded || !packageManifest}
+              >
+                <BlueArrowCircleUpIcon className="co-icon-space-r" />
+                {ocsSubscriptionLoaded ? ocsSubscription?.[0].spec.channel : null}
+              </Button>
```

Comment:
```suggestion
          
            <DetailItem
              key="version"
              title={t('ceph-storage-plugin~Version')}
              isLoading={!subscriptionLoaded}
              error={subscriptionLoaded && !ocsVersion}
            >
              {ocsVersion}
              {!_.isEmpty(filteredVersions) &&
          isManual ? (
              <Button
                type="button"
                isInline
                onClick={ocsChannelModal}
                variant="link"
                isDisabled={!ocsSubscriptionLoaded || !packageManifest}
              >
                <BlueArrowCircleUpIcon className="co-icon-space-r" />
                {ocsSubscriptionLoaded ? ocsSubscription?.[0].spec.channel : null}
              </Button>
              )
```
We want to only hide the update button and if there are no update channels we still want to show the current version. 
Use lodash helpers from `_` alias and don't import helpers directly.
Move out the `isManual` to a variable outside of the return part.

Reply:
@bipuladh As per the current code, I have hidden the update one when no new channels are available. The current version is shown when no update channel is available. We don't need to show same thing 2 times.

Reply:
![Screenshot from 2021-06-10 10-56-40](https://user-images.githubusercontent.com/54092533/121469704-954f7c80-c9da-11eb-9029-dbdf98dcee95.png)
As per the designs we want to show them separately. The Button should have the latest channel.